src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx

```diff
@@ -1,37 +1,74 @@
 import {PlatformKey} from 'app/data/platformCategories';
 import {t} from 'app/locale';
 
-const commonTermsDescription = {
-  crashes: t('Number of sessions with a crashed state'),
-  'crash-free-users': t('Number of unique users with non-crashed sessions'),
-  'crash-free-sessions': t('Number of non-crashed sessions'),
+export enum SessionTerm {
+  CRASHES = 'crashes',
+  CRASHED = 'crashed',
+  ABNORMAL = 'abnormal',
+  CRASH_FREE_USERS = 'crash-free-users',
+  CRASH_FREE_SESSIONS = 'crash-free-sessions',
+  HEALTHY = 'healthy',
+  ERRORED = 'errored',
+  UNHANDLED = 'unhandled',
+}
+
+export const sessionTerm = {
+  [SessionTerm.CRASHES]: t('Crashes'),
+  [SessionTerm.CRASHED]: t('Crashed'),
+  [SessionTerm.ABNORMAL]: t('Abnormal'),
+  [SessionTerm.CRASH_FREE_USERS]: t('Crash Free Users'),
+  [SessionTerm.CRASH_FREE_SESSIONS]: t('Crash Free Sessions'),
+  [SessionTerm.HEALTHY]: t('Healthy'),
+  [SessionTerm.ERRORED]: t('Errored'),
+  [SessionTerm.UNHANDLED]: t('Unhandled'),
 };
 
-const mobileTermsDescription = {
-  crashed: t(
+// This should never be used directly (except in tests)
+export const commonTermsDescription = {
+  [SessionTerm.CRASHES]: t('Number of sessions with a crashed state'),
+  [SessionTerm.CRASH_FREE_USERS]: t('Number of unique users with non-crashed sessions'),
+  [SessionTerm.CRASH_FREE_SESSIONS]: t('Number of non-crashed sessions'),
+};
+
+// This should never be used directly (except in tests)
+export const mobileTermsDescription = {
+  [SessionTerm.CRASHED]: t(
     'The process was terminated due to an unhandled exception or a request to the server that ended with an error'
   ),
-  'crash-free-sessions': t('Number of unique sessions that did not experience a crash'),
-  abnormal: t(
+  [SessionTerm.CRASH_FREE_SESSIONS]: t(
+    'Number of unique sessions that did not experience a crash'
```

Comment:
```suggestion
    'Number of unique sessions that did not crash'
```
Other terms don't use 'experience' is that something that should be used in the other terms too?

Reply:
I think it's fine to remove the "experience" word :)